plugins/de.cau.cs.kieler.kgraph.text.ui/src/de/cau/cs/kieler/kgraph/text/ui/contentassist/KGraphProposalProvider.java

```diff
@@ -471,13 +472,19 @@ private void valueProposal(final ContentAssistContext context,
                         break;
                     case OBJECT:
                         try {
+                        
                             proposal = "\""
-                                    + optionData.getOptionClass().newInstance().toString()
+                                    + optionData.getOptionClass().getConstructor().newInstance().toString()
```

Comment:
The deprecation warning of .newInstance() suggests to use .getDeclaredConstructor().newInstance(), I would suggest to follow that advice.